FP_Factions/gordon_reskins/cyrus/config.cpp

```diff
@@ -0,0 +1,48 @@
+class CfgPatches
+{
+	class gordon_reskins_cyrus
+	{
+		requiredAddons[]=
+		{
+			"A3_Data_F_AOW_Loadorder"
+		};
+		requiredVersion=0.1;
+		units[]={};
+		weapons[]=
+		{
+			"FP_GOR_DMR_05_pacifichex_F",
+			"FP_GOR_DMR_05_olive_F",
+			"FP_GOR_DMR_05_olive2_F"

```

Comment:
this one does not exist, forgotten to add or is this just a duplicate?

Reply:
was a testing version, forgot to remove it, fixed now